src/main/scala/org/broadinstitute/dsde/workbench/sam/config/package.scala

```diff
@@ -56,6 +56,8 @@ package object config {
       config.getString("appsDomain"),
       config.getString("pathToPem"),
       config.getString("serviceAccountClientId"),
+      config.getString("serviceAccountClientEmail"),
+      config.getString("serviceAccountClientProject"),
```

Comment:
is there a PR for these config changes?

Reply:
https://github.com/broadinstitute/firecloud-develop/pull/834
https://github.com/broadinstitute/firecloud-develop/pull/835
https://github.com/broadinstitute/firecloud-develop/pull/836